src/doc/trpl/hello-cargo.md

```diff
@@ -23,20 +23,21 @@ README][cargoreadme] for specific instructions about installing it.
 
 Let’s convert Hello World to Cargo.
 
-To Cargo-ify our project, we need to do two things: Make a `Cargo.toml`
-configuration file, and put our source file in the right place. Let's
-do that part first:
+To Cargo-ify our project, we need to do three things: Make a `Cargo.toml`
+configuration file, put our source file in the right place, and get rid of the
+old  `main.exe`. Let's do that part first:
```

Comment:
there are two spaces between 'old' and 'main.exe', and it should say something like 'main or main.exe', to cover all platforms